### PR TITLE
feat setting

### PR DESCRIPTION
**Pull request by deepak29-git**
Add setting feat
Screenshot (148)

### CODE REVIEW

File: src/Components/Setting/Setting.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+.setting-title::after {
+  content: attr(data-hover);
+  visibility: hidden;
+  color: var(--white-color);
+  font-size: 13px;
+  background-color: var(--black-color);
+  padding: 0.5rem;
+  position: relative;
+  bottom: 90px;
```
Comment (anshikaa12) [inline src/Components/Setting/Setting.css:25]: you should use rem for responsiveness

**Review by anshikaa12 — APPROVED** [review]
code is neat and well structured

File: src/Components/Setting/Setting.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.setting-title {
+  position: absolute;
+  bottom: -20px;
+  left: 30px;
```
Comment (KanekarSnehal) [inline src/Components/Setting/Setting.css:4]: you can we rem instead for px

**Review by KanekarSnehal — APPROVED** [review]
Code is well structured, you can merge it.

**Review by Mohitkumar33 — APPROVED** [review]

File: src/Components/Setting/Setting.css (hunk 1 above)
Comment (deepak29-git) [reply src/Components/Setting/Setting.css:25]: ok

File: src/Components/Setting/Setting.css (hunk 2 above)
Comment (deepak29-git) [reply src/Components/Setting/Setting.css:4]: ok

**Pull request merged**